docs/file-reference.md

```diff
@@ -528,40 +528,40 @@ services:
 volumeClaims:
 - name: database
   size: 500Mi
-  # https://kubernetes.io/docs/api-reference/v1.8/#persistentvolumeclaim-v1-core
+  # https://kubernetes.io/docs/api-reference/v1.8/#claim-v1-core
```

Comment:
wrong link, should be as it was before `https://kubernetes.io/docs/api-reference/v1.8/#persistentvolumeclaim-v1-core`